BUILD.md

```diff
@@ -59,19 +59,38 @@ You can use it by replacing the step 3 with the following:
  - If step 4 gives you a different error, you could report it to this repo or try using GCC. Just make sure your standard library and compilers are updated since Cemu uses a lot of modern features!
 - If step 4 gives you undefined libdecor_xx, you are likely experiencing an issue with sdl2 package that comes with vcpkg. Delete sdl2 from vcpkg.json in source file and recompile.
 
-## MacOS
+## macOS
 
 To compile Cemu, a recent enough compiler and STL with C++20 support is required! LLVM 13 and 
 below, built in LLVM, and Xcode LLVM don't support the C++20 feature set required. Currently, 
 LLVM 15 isn't supported due to compatibility issues with Boost dependency. The OpenGL graphics
-API isn't support on MacOS, Vulkan must be used. Additionally Vulkan must be used through the 
-Molten-VK compatibility layer.  
+API isn't support on macOS, Vulkan must be used. Additionally Vulkan must be used through the 
+Molten-VK compatibility layer
+
+### On Apple Silicon Macs, Rosetta 2 and the x86_64 version of Homebrew must be used
+
+You can skip this section if you have an Intel Mac. Every time you compile, you need to perform steps 2 & 4.
+
+1. `softwareupdate --install-rosetta` # Install Rosetta 2 if you don't have it
+2. `arch -x86_64 zsh` # run an x64 shell
+3. `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"` # install x86_64 brew. Only has to be run once
```

Comment:
This may be confusing as the same step is repeated again later on. Perhaps instead, the eval command should be moved to "Installing brew"?

Reply:
I agree, it's a bit confusing but `eval "$(/usr/local/Homebrew/bin/brew shellenv)"` needs to be run when running `brew install ...` as well as before compiling.

If you compile without it it will look in `/opt/homebrew/include` instead of `/usr/local/include` for headers and it won't compile.

<img width="660" alt="Screenshot 2022-10-05 at 12 47 01 am" src="https://user-images.githubusercontent.com/108294295/193950466-c5ca8b05-ca78-48e9-a53e-d86ed23d3008.png">

Reply:
Another note: in the instructions it says that you need to run the eval command before every compilation. On my machine, this is not the case. I just run zsh in x86 and it works.

Reply:
Updated. Thanks for your feedback.

Reply:
The optional verification stage should also be updated/removed as it now asks the user to check the brew prefix before brew is installed.

Reply:
I've removed it

Reply:
Ah, you beat me to it hah